fuse_optimizers/include/fuse_optimizers/fixed_lag_smoother_params.h

```diff
@@ -59,16 +60,23 @@ struct FixedLagSmootherParams
   ros::Duration transaction_timeout { 0.1 };
 
   /**
-   * @brief Helper function that loads strictly positive floating point values from the parameter server
+   * @brief Ceres Solver::Options object that controls various aspects of the optimizer.
+   */
+  ceres::Solver::Options solver_options;
+
+  /**
+   * @brief Helper function that loads strictly positive integral or floating point values from the parameter server
    *
    * @param[in] node_handle - The node handle used to load the parameter
    * @param[in] parameter_name - The parameter name to load
    * @param[in] default_value - A default value to use if the provided parameter name does not exist
    * @return The loaded (or default) value
    */
-  double getPositiveParam(const ros::NodeHandle& node_handle, const std::string& parameter_name, double default_value)
```

Comment:
At the end I don't really use `getPositiveParam` for integral types, but I think this change is for good, although it'd be great to have this function in a more generic place. Either way, if you don't like this, it's in its own commit, so it's easy drop it from this PR.